code/modules/mob/transform_procs.dm

```diff
@@ -55,10 +55,8 @@
 	canmove = 0
 	icon = null
 	invisibility = 101
-	for(var/t in bodyparts)
-		qdel(t)
-	for(var/i in internal_organs)
-		qdel(i)
+	QDEL_NULL(bodyparts)
+	QDEL_NULL(internal_organs)

```

Comment:
don't qdel lists; there's an other define for this.

Reply:
oh yeah whoops